src/StatsBase.jl

```diff
@@ -156,7 +156,8 @@ module StatsBase
     residuals,
     r2,
     r²,
-    model_response
+    model_response,
+    ConvergenceException
```

Comment:
I don't understand. This doesn't seem to go against the existing style, and he isn't removing an existing line break?